addons/hr_fleet/models/res_users.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields
+
+class User(models.Model):
+    _inherit = ['res.users']
+
+    employee_cars_count = fields.Integer(related='employee_id.employee_cars_count')
+
+    def __init__(self, pool, cr):
+        """ Override of __init__ to add access rights.
+            Access rights are disabled by default, but allowed
+            on some specific fields defined in self.SELF_{READ/WRITE}ABLE_FIELDS.
+        """
+        init_res = super(User, self).__init__(pool, cr)
+        # duplicate list to avoid modifying the original reference
+        type(self).SELF_READABLE_FIELDS = type(self).SELF_READABLE_FIELDS + ['employee_cars_count']
+        return init_res
+
+    def action_get_claim_report(self):
+        return self.employee_ids.action_get_claim_report()
```

Comment:
You sure the won't be any access rights issue with this kind of wrappers ? (Multi-company). 

I don't think it would be an issue with the fleet methods, but it will probably be the case for other modules.

Reply:
It should be `self.employee_id` instead of `self.employee_idS`, thanks.
I don't think there will be multi company access issues with the correction .